be/src/exec/pipeline/exchange/sink_buffer.h

```diff
@@ -26,49 +27,40 @@ struct TransmitChunkInfo {
 
 class SinkBuffer {
 public:
-    SinkBuffer(MemTracker* mem_tracker, size_t channel_number, size_t num_sinkers)
-            : _mem_tracker(mem_tracker), _num_sinkers_per_channel(channel_number, num_sinkers) {
-        for (size_t i = 0; i < channel_number; ++i) {
+    SinkBuffer(MemTracker* mem_tracker, size_t channel_num, size_t sink_num)
+            : _channel_num(channel_num),
+              _sink_num(sink_num),
+              _expected_eos_num(channel_num * sink_num),
+              _mem_tracker(mem_tracker),
+              _num_sinkers_per_channel(channel_num, sink_num) {
+        _io_threads = ExecEnv::GetInstance()->pipeline_io_thread_pool();
```

Comment:
Don't pipeline_io_thread_pool, which is used for Scan

Reply:
ok

Reply:
fixed